lib/actions-oclif/preload.ts

```diff
@@ -195,27 +185,23 @@ Can be repeated to add multiple certificates.\
 			}
 		};
 
-		options.commit = this.isCurrentCommit(options.commit || '')
+		const commit = this.isCurrentCommit(options.commit || '')
 			? 'latest'
 			: options.commit;
-		options.image = params.image;
-		options.appId = options.app;
-		delete options.app;
+		const image = params.image;
+		const appId = options.app;
 
-		options.splashImage = options['splash-image'];
-		delete options['splash-image'];
+		const splashImage = options['splash-image'];
```

Comment:
These deletes were causing problems as they're not optional and so deleting would violate the type. I changed to assign to variables and remove the "internal only" parts from FlagsDef as it better types what we actually receive and avoids this rename/delete props pattern which is harder to follow imo than straight assignments

Reply:
Right, that's much tidier.